common/templates/layouts/base.njk

```diff
@@ -149,6 +150,14 @@
       </div>
     {% endif %}
   {% endblock %}
+  
+  {% if REQUEST_PATH === '/' %}
```

Comment:
```suggestion
  {% if whatsNewContent %}
```

I think having something like this would make the template more flexible, and it makes it easier for us to show banner content on a different page if we wanted to.

Reply:
After speaking to Lizzie, we only want to show the banner on the home page and nowhere else. I guess if we need it on every page we can always refactor it to do so, but I feel for now this is okay?

Reply:
Hmm, I was thinking it feels wrong to have `REQUEST_PATH === '/'` as it makes the template quite tied to the logic of the controller, and makes an assumption that the dashboard will always be at `/`. It's not a big deal, but it feels nicer to me to not rely on the path in the template and instead rely on whether the data is available.

Reply:
Yeahh I do agree it did feel a tad `dirty` when I was doing it.